daemon/hostdaemon.go

```diff
@@ -88,6 +107,43 @@ func (d *HostDaemon) connectWithRetry() {
 	d.client = pb.NewBridgePortServiceClient(conn)
 }
 
+func (d *HostDaemon) addHandler(req *cnitypes.PodRequest) (*cni100.Result, error) {
+	pf := 0
+	vf := req.CNIConf.VFID
+	mac := req.CNIConf.MAC
+	vlan := 7
+	d.log.Info("addHandler", "pf", pf, "vf", vf, "mac", mac, "vlan", vlan)
+	err := d.CreateBridgePort(pf, vf, vlan, mac)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to call CreateBridgePort: %v", err)
+	}
+	d.log.Info("addHandler CreateBridgePort succeeded")
+
+	sm := sriov.NewSriovManager()
+	res, err := sm.CmdAdd(req)
+	if err != nil {
+		return nil, errors.New("SRIOV manager falied in add handler")
+	}
+	d.log.Info("addHandler sm.CmdAdd succeeded")
+	return res, nil
+}
+
+func (d *HostDaemon) delHandler(req *cnitypes.PodRequest) (*cni100.Result, error) {
+	pf := 0
+	vf := req.CNIConf.VFID
+	mac := req.CNIConf.MAC
+	vlan := 7
```

Comment:
We are reusing these. They should definitely be in a const.

Reply:
same as above.

@bn222 can you remove these and use values from the req instead?